src/foam/dao/PredicatedClientDAO.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license
+ * Copyright 2025 Google Inc. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.dao',
+  name: 'PredicatedClientDAO',
+  extends: 'foam.dao.ProxyDAO',
+
+  documentation: `DAO that filters delegate selects using a predicate.
+  Client side only, useful for filtering a DAO on the client side when server
+  decorators populate data on the fly since FilteredDAO filtering takes place in the MDAO.`,
```

Comment:
Maybe also document when this would be useful.

Reply:
Current useage is only for AccountDAO which is in the NP repo